lib/get-page-data.js

```diff
@@ -6,16 +6,21 @@ function getPageData(html) {
     const $ = cheerio.load(html);
     const $title = $('h1').first();
     const $subtitle = $('p').first();
+    const $hr = $('hr').last();
+    const footerContent = $hr.nextAll();
```

Comment:
doesn't this return `$footerContent` (`$` prefix), or since you can iterate it `$footerItems`?